.github/actions/upgrade-indexer/common.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+FILES_OLD="/usr/share/wazuh-indexer/plugins/opensearch-security/securityconfig"
+FILES_NEW="/etc/wazuh-indexer/opensearch-security"
+declare -A files_old
+declare -A files_new
+PACKAGE_NAME="${1}"
+MAJOR_MINOR_RELEASE=$((${2}))
+REFERENCE_VERSION=43
```

Comment:
What is the utility of the REFERENCE_VERSION variable? The following conditional is performed in the `check_version` function

```
    if [ -z "${REFERENCE_VERSION}" ]; then
        echo "Error: REFERENCE_VERSION is empty."
        exit 1
    fi
```

This condition will never be `true` if it is a hardcoded value, and in the second `if` statement you could use `-gt "43"` directly since it is not used elsewhere.

This file is loaded in the call `. ${ABSOLUTE_PATH}/common.sh` from the `upgrade-indexer.sh` file and nowhere else.

- Does the existence of that variable make sense? 
- What happens if in the `upgrade-indexer.sh` file there is another variable with the same name that has previously been initialized with another value? Would its value change?
- Does it make sense that the `upgrade-indexer.sh` file knows the `REFERENCE_VERSION` variable?

Reply:
This `REFERENCE_VERSION` variable stores the threshold version in which the securityconfig path changes. It is not the same value as the `MAJOR_MINOR` variable, which stores the major and minor of the latest stable version.

I used the `REFERENCE_VERSION` variable to avoid a magic number in the conditional but maybe is a good idea to remove it and avoid the problems that you explained. To offset the magic number, I add a comment explaining the conditional. Done in https://github.com/wazuh/wazuh-packages/commit/aa38fb6ca5a7238ffe9579e312ec77a12e6564e9